exoctk/exoctk_app/templates/phase_constraint.html

```diff
@@ -0,0 +1,120 @@
+{%extends "base.html" %}
+
+{% block content %}
+<!doctype html>
+<link rel="stylesheet" href="/static/css/select2-bootstrap-theme.min.css">
+<link rel="stylesheet" href="/static/css/icheck-bootstrap.min.css">
+<script type="text/javascript" async
+        scr="/static/js/starter.form.js">
+</script>
+<script type="text/javascript" async
+        src="/static/js/select2.min.js">
+</script>
+
+<div class='form-group'>
+    <h1>Phase Constraint Calculator</h1>
+    <br>
+
+    <p>
+        The Phase Constraint Calculator provides a simple interface for calculating the JWST observation start window.  
```

Comment:
This sounds good - assuming the info is correct, lol - once we're happy with this blurb I can also add it to the github README and the ReadTheDocs.

Reply:
Kevin sent this to me yesterday when I showed him the layout of the page, if we trust that guy putting it in the docs should be okay haha!

Reply:
ahahaha, perfect. Thanks! I'll add it over to my documentation PR so I remember to add it in other places after we've merged this.